<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personal_setting</class>
 <widget class="QDialog" name="personal_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 188, 46);</string>
  </property>
  <widget class="QGraphicsView" name="foreground">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>600</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 40);
border-radius:  20px; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_signup">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>40</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>231</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  10px; 
background-color: rgb(255, 255, 255);

color: rgb(127, 127, 127);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  10px; 
background-color: rgb(255, 255, 255);

color: rgb(127, 127, 127);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>PHONE</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_gender">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>231</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  10px; 
background-color: rgb(255, 255, 255);
color: rgb(127, 127, 127);</string>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit_birthday">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>231</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  10px; 
background-color: rgb(255, 255, 255);

color: rgb(127, 127, 127);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>320</y>
     <width>51</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>+886|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/back arrow.png);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_alert">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Consolas&quot;;
color: rgb(64, 64, 64);
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border: 3px;

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_finish">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
border: 2px solid white;
border-radius:  15px; </string>
   </property>
   <property name="text">
    <string>FINISH</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>comboBox_gender</tabstop>
  <tabstop>dateEdit_birthday</tabstop>
  <tabstop>foreground</tabstop>
  <tabstop>pushButton_back</tabstop>
  <tabstop>textBrowser_alert</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
